api/proto/dop/guide/guide.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto3";
+
+package erda.dop.guide;
+option go_package = "github.com/erda-project/erda-proto-go/dop/guide/pb";
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "common/openapi.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+service GuideService {
+  option(erda.common.openapi_service) = {
+    service : "dop",
+    auth : {
+      check_login : true,
+      check_token : true,
+    }
+  };
+
+  rpc CreateGuideByGittarHook(GittarPushPayloadEvent)
+      returns(CreateGuideResponse) {
+    option(google.api.http) = {
+      post : "/api/guide/actions/create-by-gittar-hook",
+    };
+    option(erda.common.openapi) = {
+      path : "/api/guide/actions/create-by-gittar-hook",
+    };
+  }
```

Comment:
we should define a common hook api

Reply:
The structure of each hook definition is different.
And now clear interface definition.